docs/api/methods/js_kanban_settheme_method.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: setTheme()
+title: setTheme Method
+description: You can learn about the setTheme method in the documentation of the DHTMLX JavaScript Kanban library. Browse developer guides and API reference, try out code examples and live demos, and download a free 30-day evaluation version of DHTMLX Kanban.
+---
+
+# setTheme()
+
+### Description
+
+@short: Applies a new theme to Kanban dynamically (with reinitialization)
+
+### Usage
+
+~~~jsx {}
+setTheme(theme?: string): void;
```

Comment:
theme: TThemeConfig

```js
type TThemeConfig = {
	name: string;
	fonts: boolean;
};
```
it's the same as `theme` in Kanban props, you have already documented it (have you?)